acq4/modules/AutomationDebug/cell.py

```diff
@@ -26,27 +27,27 @@ def __init__(self, position):
         self._positions = {ptime.time(): position}
         self._imager = None
         self._trackingFuture = None
-        self.isTracking = False
         self._tracker = None
-        self._roiSize = None
 
     @property
-    def position(self):
-        """Get the current position of the cell."""
-        return np.array(self._positions[max(self._positions)])
+    def isTracking(self):
```

Comment:
Can all visual tracking API like isTracking and the signals above just be reached from `cell.tracker` ?